### PR TITLE
Add goss ansible role

### PR DESCRIPTION
**Pull request by tei-k**
# Description
https://scalar-labs.atlassian.net/browse/DLT-8192

# Done
- Add goss role for test server configuration.

# How to use
- Output `ssh.cfg` from network module.
```
$ terraform output ssh_config > ssh.cfg
```
- Output inventory file
```
$ terraform output inventory_ini > cassandra.ini
```
- Execute test for cassandra server
```
$ ansible-playbook --ssh-common-args="-F ssh.cfg" -i cassandra.ini ./provision/ansible/playbooks/goss-server.yaml -l cassandra
```

### CODE REVIEW

File: provision/ansible/playbooks/roles/goss/files/config_vars/aws_bai.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+bastion:
```
Comment (ymorimo) [inline provision/ansible/playbooks/roles/goss/files/config_vars/aws_bai.yaml:1]: config_vars files are currently only for bai and default?

**Review by ymorimo — COMMENTED** [review]
The tests are assumed to be run manually?

**Comment by ymorimo** [comment]
> The tests are assumed to be run manually?

Sorry, but I found that scalar-labs/scalar-terratest#12 contains the script.

File: provision/ansible/playbooks/roles/goss/files/config_vars/aws_bai.yaml (hunk 1 above)
Comment (tei-k) [reply provision/ansible/playbooks/roles/goss/files/config_vars/aws_bai.yaml:1]: Yes, I'll create new PR for chiku and sho.

**Comment by tei-k** [comment]
>>The tests are assumed to be run manually?

>Sorry, but I found that scalar-labs/scalar-terratest#12 contains the script.

Currently, this goss role is running in terratest or manually.

**Review by ymorimo — APPROVED** [review]
LGTM!

**Review by ymorimo — COMMENTED** [review]
Question: Since Goss is run from scalar-terratest, should the role be added to this repo?

**Comment by tei-k** [comment]
>Question: Since Goss is run from scalar-terratest, should the role be added to this repo?

Good point! 👍 
Since it's used for testing `scalar-terraform`, I think it's more appropriate to put it in the repo here.

**Review by feeblefakie — APPROVED** [review]
LGTM!

**Review by ymorimo — APPROVED** [review]
LGTM!

**Pull request merged**